src/components/deck/index.js

```diff
@@ -92,7 +92,7 @@ const Deck = ({
   }, [filteredChildren]);
 
   // Initialise useDeck hook and get state and dispatch off of it
-  const { state, dispatch } = useDeck(initialState);
+  const { state, dispatch } = useDeck({ ...initialState, numberOfSlides });
```

Comment:
Why is `numberOfSlides` getting passed to `useDeck`?

Reply:
Nvm, I see why. If we want to keep`numberOfSlides` in deck context, can you unify that usage across the app? Some places are getting it via props, some via context.

For instance, https://github.com/FormidableLabs/spectacle/blob/db738e7ec5ab86d8be5f585f5c32edc12a5079c7/src/components/deck/index.js#L178.

Instead, we can proxy `numberOfSlides` through `useSlide` instead of copying it into props as in that link.